molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/VcfValidator.java

```diff
@@ -0,0 +1,106 @@
+package org.molgenis.data.annotation.cmd;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class VcfValidator
+{
+	private String perlLocation;
+	private String workingDirectory;
+	private String vcfToolsDirectory;
+
+	public VcfValidator(String perlLocation, String vcfToolsDirectory)
+	{
+		this.perlLocation = perlLocation;
+		this.vcfToolsDirectory = vcfToolsDirectory;
+	}
+
+	/**
+	 * Validation method that calls the perl executable of the vcf-validator. Logs vcf validation output into a log file
+	 * 
+	 * @param vcfFile
+	 * @return Message
+	 */
+	public String validateVCF(File vcfFile)
+	{
+		try
+		{
+			String vcfValidator = vcfToolsDirectory + "perl/vcf-validator";
+			// Checks if vcf-tools is present
+			if (new File(vcfValidator).exists())
+			{
+				// Set working directory, Vcf.pm should be built here
+				workingDirectory = vcfToolsDirectory + "perl/";
+
+				ProcessBuilder processBuilder = new ProcessBuilder(perlLocation, vcfValidator,
+						vcfFile.getAbsolutePath(), "-u", "-d").directory(new File(workingDirectory));
+
+				Process proc = processBuilder.start();
+
+				InputStream inputStream = proc.getInputStream();
+				Scanner scanner = new Scanner(inputStream);
+
+				String line = "";
+				Integer errorCount = null;
+				Pattern p = Pattern.compile("(\\d+)\\s*errors\\s*total");
+
+				File logFile = new File("vcf-validation.log");
```

Comment:
Maybe use the output file's name here, so that the validation logs don't overlap